.github/workflows/neurips23.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           - algorithm: vamana
             dataset: random-xs
             track: ood
-          - algorithm: sustech-ood
```

Comment:
please rebase add a separate entry for your algo

Reply:
sorry, fixed in 17a568a